hr_employee_calendar_planning/models/hr_employee.py

```diff
@@ -104,8 +104,9 @@ def _regenerate_calendar(self):
                         "active": False,
                         "company_id": self.company_id.id,
                         "auto_generate": True,
-                        "name": _("Auto generated calendar for employee")
-                        + " %s" % self.name,
+                        "name": _("Auto generated calendar for employee {}").format(
```

Comment:
Please don't change this, as we don't want to have the `{}` translatable, as then it can be easily mangled by a translator.